NeuroID/NeuroIDClass/NeuroID.swift

```diff
@@ -128,6 +128,11 @@ public class NeuroID: NSObject {
         networkMonitor?.startMonitoring()
 
         captureApplicationMetaData()
+        
+        let logEvent = NIDEvent(type: .log)
+        logEvent.m = "isAdvancedDevice setting: \(isAdvancedDevice)"
+        logEvent.level = "INFO"
+        NeuroID.saveEventToDataStore(logEvent)
```

Comment:
NIT: we have a log message event init

NIDEvent(type:.log, level:String, m:String)

Reply:
using suggested method signature